distribution/ddf-common/src/main/resources-filtered/etc/startup.properties

```diff
@@ -1,14 +1,14 @@
 # Bundles to be started on startup, with startlevel
-mvn\:org.apache.karaf.features/org.apache.karaf.features.extension/4.1.2 = 1
-mvn\:org.apache.felix/org.apache.felix.metatype/1.1.2 = 5
-mvn\:org.apache.karaf.services/org.apache.karaf.services.eventadmin/4.1.2 = 5
-mvn\:org.ops4j.pax.url/pax-url-aether/2.5.2 = 5
+mvn\:org.apache.karaf.features/org.apache.karaf.features.extension/4.1.4 = 1
+mvn\:org.apache.felix/org.apache.felix.metatype/1.1.6 = 5
+mvn\:org.apache.karaf.services/org.apache.karaf.services.eventadmin/4.1.4 = 5
+mvn\:org.ops4j.pax.url/pax-url-aether/2.5.3 = 5
 mvn\:org.ops4j.pax.logging/pax-logging-api/1.10.1 = 8
 mvn\:org.ops4j.pax.logging/pax-logging-log4j2/1.10.1 = 8
 # Using a boot feature won't be good enough. We need to actually replace Felix with our own impl.
 # The following ddf.platform.osgi lines (for the currently used karaf version) used to be:
 # mvn\:org.apache.felix/org.apache.felix.configadmin/1.8.14 = 10
 mvn\:ddf.platform.osgi/platform-osgi-internal-api/${project.version} = 9
```

Comment:
@Lambeaux Will we need to do anything to these two customized libraries to support the upgrade?

Reply:
@coyotesqrl No. As long as we keep the version of felix configadmin constant we will be fine.